File: backend/models/rfidcard.go
package models

import "time"

type RfidCard struct {
	ID             uint      `json:"id"`
	RfidTag        string    `json:"rfid_tag"`
	ActivationDate time.Time `json:"activation_date"`
	Status         int8      `json:"status"`
	EmployeeID     uint      `json:"employee_id"`
}

File: frontend/src/api/rfid_card.ts
import { RfidCard } from "@/types/rfid_cards";

export async function getRfidCards(): Promise<RfidCard[]> {
    return Promise.resolve([
        {
            id: 1,
            rfid_tag: '123456',
            employee_id: 1,
            employee_name: 'John Doe',
            activation_date: '2023-01-01',
            status: 1
        }
    ])
}